include/tpm2-tss-engine.h

```diff
@@ -52,7 +52,8 @@ typedef struct {
 } TPM2_DATA;
 
 #define TPM2TSS_SET_OWNERAUTH   ENGINE_CMD_BASE
-#define TPM2TSS_SET_TCTI        (ENGINE_CMD_BASE + 1)
+#define TPM2TSS_SET_PARENTAUTH  (ENGINE_CMD_BASE + 1)
+#define TPM2TSS_SET_TCTI        (ENGINE_CMD_BASE + 2)
```

Comment:
Please don't change the ABI in incompatible ways; i.e. keep the old numbers as they were and just add to the bottom...

Reply:
Good point, I actually meant to change this before making the PR but it clearly slipped my mind. I'll make the fix.